website/source/guides/identity/identity.html.md

```diff
@@ -0,0 +1,1127 @@
+    $ tee payload-group.json <<<EOF
+    {
+      "name": "engineers",
+      "policies": ["team-eng"],
+      "member_entity_ids": ["<ENTITY_ID>"],
+      "metadata": {
+        "team": "Engineering",
+        "region": "North America"
+      }
+    }
+    EOF
+
+    # Use identity/group endpoint
+    $ curl --header "X-Vault-Token: ..." \
+           --request PUT \
+           --data @payload-group.json \
+           http://127.0.0.1:8200/v1/identity/group | jq
+    {
+       "request_id": "2b6eefd6-67a6-31c7-dbc3-11c1c132e2cf",
+       "lease_id": "",
+       "renewable": false,
+       "lease_duration": 0,
+       "data": {
+         "id": "d62157aa-b5f6-b6fe-aa40-0ffc54defc41",
+         "name": "engineers"
+       },
+       ...
+    ```
+
+Now, when you login as `bob` or `bsmith`, its generated token inherits the
+group-level policy, **`team-eng`**. You can perform similar tests demonstrated
+in [Step 3](#step3) to verify that.
+
+
+#### Web UI
+
+1. Click the **Policies** tab, and then select **Create ACL policy**.
+
+1. Enter **`team-eng`** in the **Name** field, and paste in the [`team-eng.hcl` policy
+rules](#step3) in the **Policy** text editor, and then click **Create Policy**.
+
+1. Click the **Access** tab and select **Entities**.
+
+1. Select the **`bob-smith`** entity and copy its **ID** displayed under the
+**Details** tab.
+
+1. Now, click **Groups** from the left navigation, and select **Create group**.
+
+1. Enter the group information as shown below.
+
+    ![Group](/assets/images/vault-entity-7.png)
+
+    ~> **NOTE:** Make sure to enter the `bob-smith` entity **ID** you copied in the
+    **Member Entity IDs** field.
+
+1. Click **Create**.
+
+Now, when you login as `bob` or `bsmith`, its generated token inherits the
+group-level policy, **`team-eng`**. You can perform similar tests demonstrated
+in [Step 3](#step3) to verify that.
+
+<br>
+
+> **Summary:** By default, Vault creates an internal group. When you create an
+internal group, you specify the ***group members*** rather than ***group
+alias***. Group _aliases_ are mapping between Vault and external identity providers
+(e.g. LDAP, GitHub, etc.).  Therefore, you define group aliases only when you
+create **external** groups.  For internal groups, you specify `member_entity_ids`
+and/or `member_group_ids`.
+
+
+
+### <a name="step4"></a>Step 4: Create an External Group
+
+It is common for organizations to enable auth methods such as LDAP, Okta and
+perhaps GitHub to handle the Vault user authentication. A user may belong to one
+or more groups defined within the identity provider. Also, the Vault server may
+have one or more auth methods enabled.
+
+In order to manage the group-level authorization, you can create an external
+group to link Vault with the external identity provider (auth provider).
+
+#### Example Scenario
+
+Any user who belongs to **`certification`** and **`training`** teams in GitHub
```

Comment:
This scenario is not handled by Vault. One team can map to one external group in Vault. An external group in Vault can only have one group-alias. If you try to assign multiple group aliases to an external group, it will only override the previous one.

When a team in GH is mapped to external group in Vault, entity ID of the user will automatically be added to the external group upon login. Any change in user's membership to the team will reflect during renewals and/or re-logins.